### ISSUE TITLE
Bug: Season Switch and Conferences in fake phases

### ISSUE THREAD

**Issue by F3PiX** [rails-girls-summer-of-code/rgsoc-teams#574]
When switching to the fake application phase or the fake proposal phase in /orga/seasons
and clicking on Community, 
I get a 500 with this message:
```
[Rollbar] Reporting exception: PG::UndefinedTable: ERROR:  missing FROM-clause entry for table "conferences"
LINE 1: ... WHERE (teams.name ILIKE '%%') GROUP BY users.id, conference...
                                                             ^
```

It happens only in dev mode.

**Comment by alicetragedy** [comment]
Update: doesn't just happen in dev mode, it's an issue currently on the production site as well.

**Comment by carpodaster** [comment]
Fixed in #584 – thanks @F3PiX !
